azure-functions-powershell-library/GetFunctionsMetadataCommand.cs

```diff
@@ -0,0 +1,48 @@
+﻿//
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+//
+
+using Common;
+using Microsoft.Azure.Functions.PowerShellWorker;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Functions.PowerShell
+{
+    [Cmdlet(VerbsCommon.Get, "FunctionsMetadata")]
+    public class GetFunctionsMetadataCommand : Cmdlet
+    {
+        /// <summary>
+        /// The path of the Azure Functions directory
+        /// </summary>
+        [Parameter(Mandatory = true, Position = 0)]
+        public string? FunctionsAppDirectory { get; set; }
+
+        private string outputJson { get; set; } = string.Empty;
+
+        protected override void ProcessRecord()
+        {
+            try
+            {
+                if (FunctionsAppDirectory != null && !string.IsNullOrEmpty(FunctionsAppDirectory) && Directory.Exists(FunctionsAppDirectory))
```

Comment:
Does `FunctionsAppDirectory` need to be a nullable string in line 19? I believe that as a reference type, it can already be `null` by default, in which case this check needs only to be `!string.IsNullOrEmpty(FunctionsAppDirectory) && Directory.Exists(FunctionsAppDirectory)`
```suggestion
                if (!string.IsNullOrEmpty(FunctionsAppDirectory) && Directory.Exists(FunctionsAppDirectory))
```